src/crypto/SecretStorage.js

```diff
@@ -207,24 +438,13 @@ export class SecretStorage extends EventEmitter {
                 throw new Error("Unknown key: " + keyId);
             }
 
-            // check signature of key info
```

Comment:
Hmm, so we don't need to verify the 4S key is trusted when storing secrets now...?

Reply:
Yes.  We know the key is trusted because the user entered the secret key.